packages/cmf/package.json

```diff
@@ -93,7 +93,8 @@
     "redux-storage-decorator-filter": "^1.1.8",
     "redux-storage-decorator-immutablejs": "^1.0.4",
     "redux-storage-engine-localstorage": "^1.1.4",
-    "redux-thunk": "^2.2.0"
+    "redux-thunk": "^2.2.0",
+    "uuid": "^3.3.0"
```

Comment:
When you add a lib please check versions.js first 

```javascript
uuid: '^3.0.1', // prefer bson-objectid
```

So please replace it with bson-objectid